app.js

```diff
@@ -1,7 +1,15 @@
 const express = require('express');
-const { getTopics } = require('./controllers/news.controllers');
+const { getTopics, getArticles, getArticle } = require('./controllers/news.controllers');
 const app = express();
 
 app.get('/api/topics', getTopics)
+app.get('/api/articles', getArticles)
+app.get('/api/articles/:article_id', getArticle)
+
+app.use((err, req, res, next) => {
+  if (err.status && err.msg) res.status(err.status).send({msg: err.msg})
+	else if (err = '22P02') res.status(400).send({ msg: "Doesn't exist"})
+	else res.status(500).send('Server Error!');
```

Comment:
At some point in the near future this single error handling middleware you've got is going to get quite cluttered with `if` statement logic - I'd recommend splitting it out into a few different error handlers, one for `custom errors` one for `psql` errors, one for `500` etc. making use of `next(err)` to pass the err along them - perhaps on the next ticket